src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,12 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Return true if the command mutates the addressbook data
+     * Set to return false for parent class Command and override subclasses that mutates command to return true
```

Comment:
The second line is redundant. You can remove it as the first line already gives the information on how to override it in subclasses. Meanwhile, the parent class should not be aware of any subclasses at the point of its creation. To make the comment more precise, you can think about whether the command is always mutating when this method return true. For example, it will return true for an add command. However, when you add a duplicate person to the unique list, is this add operation truly mutating? Therefore, using "return true ...... potentially mutates ......" would be better.